dynatrace-oneagent/deploy-dynatrace-oneagent.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 set -e
 
+PLATFORM=$1
+if [ "$PLATFORM" = "KUBERNETES" ]; then
+  ONEAGENT_CRD="https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/kubernetes.yaml"
+elif [ "$PLATFORM" = "OPENSHIFT" ]; then
+  ONEAGENT_CRD="https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/download/v0.8.2/openshift.yaml"
```

Comment:
i think we can also point to `latest` here to be consistent among platforms

Reply:
We should never point to `latest`, to be honest. With a given Keptn release we should make sure to also point to a oneagent operator release that we know works with our installation.
So for Kubernetes, we could use 0.9.2. 

We should also add a link to the supported platforms: https://github.com/Dynatrace/dynatrace-oneagent-operator#supported-platforms